### PR TITLE
Fine tune

### PR DESCRIPTION
**Pull request by h0ngxuanli**

### CODE REVIEW

File: README.md
Comment (zihanxing) [inline README.md:1]: remove this file

**Review by zihanxing — DISMISSED** [review]
Plz clean up the json files. It's a bit messy

**Comment by rishabhshah13** [comment]
- Data is a bit noisy
- Try to remove any unnecessary/unused training file

**Comment by h0ngxuanli** [comment]
- Resolve the conficts
- Remove duplicate data generation code

**Pull request merged**